src/js/content_scripts/input-mapper.js

```diff
@@ -0,0 +1,128 @@
+/*
+Copyright (c) 2020 The Gamepad Navigator Authors
+See the AUTHORS.md file at the top-level directory of this distribution and at
+https://github.com/fluid-lab/gamepad-navigator/raw/master/AUTHORS.md.
+
+Licensed under the BSD 3-Clause License. You may not use this file except in
+compliance with this License.
+
+You may obtain a copy of the BSD 3-Clause License at
+https://github.com/fluid-lab/gamepad-navigator/blob/master/LICENSE
+*/
+
+/* eslint-env browser */
+
+(function (fluid) {
+    "use strict";
+
+    var gamepad = fluid.registerNamespace("gamepad");
+    fluid.registerNamespace("gamepad.inputMapperUtils");
+
+    fluid.defaults("gamepad.inputMapper", {
+        gradeNames: ["gamepad.navigator"],
+        model: {
+            actionMap: {
+                buttons: {
+                    "0": "vacant",
```

Comment:
I guess you're thinking that you'd have separate maps for the action and any additional options, but to me it makes more sense to have an object that can be used to control the action and also specify non-default values relevant to the action.  For example, you might want to indicate that a thumbstick is to be used to scroll, which axis, how quickly, and whether to invert the direction of the scrolling.  It seems easier to do this with a single object per control rather than separate maps.

Reply:
Hi @the-t-in-rtf, thank you for the feedback. I haven't really thought of separate maps and I named it to `actionMaps` as that's the only option we are dealing with as of now.

I'll change it as you suggested, however, I am slightly confused here. I just need a heads-up if what I understood is correct. Did you mean that each button/axes should be an object and would contain options such as `defaultAction` and `currentAction` (for reconfigured value)? Also, if this is correct, what other options would you want me to include?

Reply:
Nevermind, I think I got the idea. But I am not exactly sure if what I've done is exactly what you meant here. I guess you would have to review my next commit to see if that's correct.